docs/index.d.ts

```diff
@@ -753,7 +753,7 @@ export interface RosettaConstructionMetadataResponse {
     recent_block_hash?: string;
     [k: string]: unknown | undefined;
   };
-  suggested_fee?: RosettaAmount;
+  suggested_fee?: RosettaAmount[];
```

Comment:
I think in this case we wouldn't need an array for `suggested_fee`, since it's only needed if the fee must be paid in multiple currencies.

Reply:
I think it is more for consistency. They declared it as an array in the spec too : https://www.rosetta-api.org/docs/models/ConstructionMetadataResponse.html